test-core.ini

```diff
@@ -90,6 +90,8 @@ ckan.tracking_enabled = true
 beaker.session.key = ckan
 beaker.session.secret = This_is_a_secret_or_is_it
 
+ckan.csrf_protection.field_name = _csrf_token
+
```

Comment:
Is this needed? Since we are also doing:
```
    # exempt the CKAN TESTS from csrf-validation.
    config["WTF_CSRF_METHODS"] = []
```

If it is needed, can we document why?

Reply:
Not anymore, I deleted it